### PR TITLE
Add Privacy Policy & ToS

### LINKED ISSUE #40: Add Privacy Policy And Terms Of Service
- [ ] Privacy Policy
- [ ] Terms of Service

### PR DESCRIPTION
**Pull request by apbendi**
Closes #40

### CODE REVIEW

File: frontend/src/components/Footer.tsx [hunk 1]
```diff
@@ -1,10 +1,36 @@
...
+      <a href="/static/docs/Zeneth-Privacy-Policy-061621.pdf" target="_blankd" className="text-purple-800">
+        Privacy Policy
+      </a>{' '}
+      |{' '}
+      <a href="/static/docs/Zeneth-Terms-of-Service-061621.pdf" target="_blankd" className="text-purple-800">
+        Terms of Service
+      </a>{' '}
+      |{' '}
+      <a href="mailto:support@zeneth.app" className="text-purple-800">
+        support@zeneth.app
```
Comment (mds1) [inline frontend/src/components/Footer.tsx:19]: Is this email setup?

Comment (mds1) [reply frontend/src/components/Footer.tsx:19]: Also, should we just use @scopelift.co emails here and for Umbra? That way it's consistent between apps and simpler to manage as we have more apps

File: frontend/src/pages/tos.tsx [hunk 2]
```diff
@@ -0,0 +1,129 @@
+const TOS = () => {
```
Comment (mds1) [inline frontend/src/pages/tos.tsx:1]: I think we can delete this file since we link to the PDFs right?

**Review by mds1 — COMMENTED** [review]
Looks good + links work! Just some small comments

File: frontend/src/components/Footer.tsx (hunk 1 above)
Comment (apbendi) [reply frontend/src/components/Footer.tsx:19]: Yes, the email is set up!  They're just forwards so managing is not too bad!

File: frontend/src/pages/tos.tsx (hunk 2 above)
Comment (apbendi) [reply frontend/src/pages/tos.tsx]: Good catch! Fixed with fe14d32

**Pull request merged**